src/Services/Item/Item.BusinessLogic/Models/Common/PagedList.cs

```diff
@@ -0,0 +1,41 @@
+﻿using Microsoft.EntityFrameworkCore;
+
+namespace Item.BusinessLogic.Models.Common;
+
+public class PagedList<T>
+{
+    private PagedList(List<T> items, int page, int pageSize, int totalCount)
+    {
+        Items = items;
+        Page = page;
+        PageSize = pageSize;
+        TotalCount = totalCount;
+        TotalPages = (int)Math.Ceiling(TotalCount / (double)pageSize);
+    }
```

Comment:
When describing a class, use the following structure:
· Declare all class members, including readonly or constants
· Declare all public properties, including auto-properties
· Declare all constructor(s)
· Declare the destructor, if any
· Declare all public methods
· Declare all protected methods
· Declare all private methods